### PR TITLE
Remove method with definite value for simplicity

### PR DESCRIPTION
**Pull request by jamesgua**
those methods return false and not referenced in other places except local, also no keyword virtual

Signed-off-by: James Guan jamesgua@ca.ibm.com

### CODE REVIEW

**Comment by andrewcraik** [comment]
@cathyzhyi can you take a look - I don't think these methods are safe to remove due to downstream use in OpenJ9 and desire to keep the code consistent while a new inliner is being built which is more modular and better able to isolate some of these things...

File: compiler/optimizer/Inliner.cpp [hunk 1]
```diff
@@ -493,12 +493,6 @@ TR_InlinerBase::cleanup(TR::ResolvedMethodSymbol * callerSymbol, bool inlinedSit
 
    }
 
-bool
-OMR_InlinerPolicy::mustBeInlinedEvenInDebug(TR_ResolvedMethod * calleeMethod, TR::TreeTop *callNodeTreeTop)
```
Comment (cathyzhyi) [inline compiler/optimizer/Inliner.cpp:497]: This one is used by downstream projects and can't be deleted.

Comment (cathyzhyi) [inline compiler/optimizer/Inliner.cpp:497]: This one is used in the downstream project and can't be deleted.

File: compiler/optimizer/Inliner.hpp [hunk 2]
```diff
@@ -651,7 +648,6 @@ class OMR_InlinerPolicy : public TR::OptimizationPolicy, public OMR_InlinerHelpe
       virtual void determineInliningHeuristic(TR::ResolvedMethodSymbol *callerSymbol);
       bool tryToInlineGeneral(TR_CallTarget *, TR_CallStack *, bool);
       virtual bool callMustBeInlined(TR_CallTarget *calltarget);
-      bool mustBeInlinedEvenInDebug(TR_ResolvedMethod * calleeMethod, TR::TreeTop *callNodeTreeTop);
```
Comment (cathyzhyi) [inline compiler/optimizer/Inliner.hpp:654]: this needs to be kept as well.

File: compiler/optimizer/Inliner.cpp (hunk 1 above)
Comment (jamesgua) [reply compiler/optimizer/Inliner.cpp]: commit added per your comments @cathyzhyi 
4c3f9bf71d5f963d6d8455e56a554b5035df3023

**Comment by cathyzhyi** [comment]
@jamesgua Hi James, could you squash all the commits into one? then I can approve the change. Thanks!

**Comment by jamesgua** [comment]
Looks like I messed up when trying to squash those commits, so close this one. In the mean time, will include a few more meaningless methods like these and create a new PR. 
Sorry for the duplicates!

**Pull request closed without merging**